### PR TITLE
Ensure relative links have a starting slash

### PR DESCRIPTION
**Pull request by queengooborg**
This PR fixes all relative links to ensure that they start with a forward slash.

### CODE REVIEW

File: files/es/web/api/canvas_api/tutorial/basic_usage/index.md [hunk 1]
```diff
@@ -18,9 +18,9 @@ A primera vista, un elemento {{HTMLElement("canvas")}} es parecido al elemento {
 
 > **Nota:** Si su renderizado se ve distorsionado, pruebe especificar los atributos width y height explícitamente en los atributos del `<canvas>` , y no usando CSS.
 
-El atributo [id](/es/docs/Web/HTML/Atributos_Globales/id) no está especificado para el elemento `<canvas>` pero es uno de los [atributos globales de HTML](/es/docs/Web/HTML/Atributos_Globales) el cual puede ser aplicado a cualquier elemento HTML (como [class](en-US/docs/Web/HTML/Global_attributes/class) por ejemplo). Siempre es buena idea proporcionar un `id` porque esto hace más fácil identificarlo en un script.
+El atributo [id](/es/docs/Web/HTML/Atributos_Globales/id) no está especificado para el elemento `<canvas>` pero es uno de los [atributos globales de HTML](/es/docs/Web/HTML/Atributos_Globales) el cual puede ser aplicado a cualquier elemento HTML (como [class](/en-US/docs/Web/HTML/Global_attributes/class) por ejemplo). Siempre es buena idea proporcionar un `id` porque esto hace más fácil identificarlo en un script.
```
Comment (Graywolf9) [inline files/es/web/api/canvas_api/tutorial/basic_usage/index.md:21]:
```suggestion
El atributo [id](/es/docs/Web/HTML/Atributos_Globales/id) no está especificado para el elemento `<canvas>` pero es uno de los [atributos globales de HTML](/es/docs/Web/HTML/Atributos_Globales) el cual puede ser aplicado a cualquier elemento HTML (como [class](/es/docs/Web/HTML/Global_attributes/class) por ejemplo). Siempre es buena idea proporcionar un `id` porque esto hace más fácil identificarlo en un script.
```

File: files/es/web/api/canvas_api/tutorial/basic_usage/index.md [hunk 2]
```diff
@@ -18,9 +18,9 @@ A primera vista, un elemento {{HTMLElement("canvas")}} es parecido al elemento {
...
 
-El elemento `<canvas>` puede ser estilizado como a cualquier imagen normal (margin, border, background, etc). Estas reglas, sin embargo, no afectan a lo dibujado sobre el canvas. Mas adelante veremos cómo se hace esto en un [capítulo dedicado](en-US/docs/Web/API/Canvas_API/Tutorial/Applying_styles_and_colors) en este tutorial. Cuando no tenemos reglas de estilo aplicadas al canvas, este será completamente transparente.
+El elemento `<canvas>` puede ser estilizado como a cualquier imagen normal (margin, border, background, etc). Estas reglas, sin embargo, no afectan a lo dibujado sobre el canvas. Mas adelante veremos cómo se hace esto en un [capítulo dedicado](/en-US/docs/Web/API/Canvas_API/Tutorial/Applying_styles_and_colors) en este tutorial. Cuando no tenemos reglas de estilo aplicadas al canvas, este será completamente transparente.
```
Comment (Graywolf9) [inline files/es/web/api/canvas_api/tutorial/basic_usage/index.md:23]:
```suggestion
El elemento `<canvas>` puede ser estilizado como a cualquier imagen normal (margin, border, background, etc). Estas reglas, sin embargo, no afectan a lo dibujado sobre el canvas. Mas adelante veremos cómo se hace esto en un [capítulo dedicado](/es/docs/Web/API/Canvas_API/Tutorial/Applying_styles_and_colors) en este tutorial. Cuando no tenemos reglas de estilo aplicadas al canvas, este será completamente transparente.
```

File: files/es/web/api/indexeddb_api/index.md [hunk 3]
```diff
@@ -32,7 +32,7 @@ Los métodos del API Asíncrono, retornan sin bloquear el hilo de llamada. Para
...
+- [`IDBVersionChangeRequest`](/en-US/docs/IndexedDB/IDBVersionChangeRequest) representa una solicitud para cambiar la versión de una base de datos. La manera de cambiar la versión de la base de datos ahora es diferente (llamando [`IDBFactory.open()`](/es/docs/IndexedDB/IDBFactory#open) sin llamar también a [`IDBDatabase.setVersion()`](</es/docs/IndexedDB/IDBDatabase#setVersion()> "/en-US/docs/IndexedDB/IDBDatabase#setVersion()")), y la interfaz [`IDBOpenDBRequest`](/es/docs/IndexedDB/IDBOpenDBRequest) tiene ahora la funcionalidad de la eliminada `IDBVersionChangeRequest`.
 - [`IDBDatabaseException`](/en-US/docs/IndexedDB/IDBDatabaseException){{deprecated_inline}} representa las condiciones de excepción que se pueden encontrar mientras se ejecutan operaciones en la base de datos.
```
Comment (Graywolf9) [inline files/es/web/api/indexeddb_api/index.md:36]:
```suggestion
- [`IDBVersionChangeRequest`](/es/docs/IndexedDB/IDBVersionChangeRequest) representa una solicitud para cambiar la versión de una base de datos. La manera de cambiar la versión de la base de datos ahora es diferente (llamando [`IDBFactory.open()`](/es/docs/IndexedDB/IDBFactory#open) sin llamar también a [`IDBDatabase.setVersion()`](</es/docs/IndexedDB/IDBDatabase#setVersion()> "/es/docs/IndexedDB/IDBDatabase#setVersion()")), y la interfaz [`IDBOpenDBRequest`](/es/docs/IndexedDB/IDBOpenDBRequest) tiene ahora la funcionalidad de la eliminada `IDBVersionChangeRequest`.
- [`IDBDatabaseException`](/es/docs/IndexedDB/IDBDatabaseException){{deprecated_inline}} representa las condiciones de excepción que se pueden encontrar mientras se ejecutan operaciones en la base de datos.
```

**Review by Graywolf9 — APPROVED** [review]
Thank you so much! I notice some urls pointing to `en-US` so heres is the correction

File: files/zh-cn/web/css/_doublecolon_cue/index.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ slug: Web/CSS/::cue
 
 {{CSSRef}}
 
-::cue CSS [伪元素](/zh-CN/docs/Web/CSS/Pseudo-elements)匹配所选元素中的[WebVTT](en-US/docs/Web/API/WebVTT_API)提示。这可以用于在 VTT 轨道的媒体中使用字幕和其他线索。
+::cue CSS [伪元素](/zh-CN/docs/Web/CSS/Pseudo-elements)匹配所选元素中的[WebVTT](/en-US/docs/Web/API/WebVTT_API)提示。这可以用于在 VTT 轨道的媒体中使用字幕和其他线索。
```
Comment (yin1999) [inline files/zh-cn/web/css/_doublecolon_cue/index.md:8]: Let's point to the localized document.


```suggestion
::cue CSS [伪元素](/zh-CN/docs/Web/CSS/Pseudo-elements)匹配所选元素中的[WebVTT](/zh-CN/docs/Web/API/WebVTT_API)提示。这可以用于在 VTT 轨道的媒体中使用字幕和其他线索。
```

File: files/zh-tw/web/api/web_workers_api/using_web_workers/index.md [hunk 5]
```diff
@@ -319,7 +319,7 @@ var asyncEval = (function () {
 })();
 ```
 
-[data URL](en-US/docs/Web/HTTP/data_URIs) 相當於網路請求，範例中的 data URL 會在 worker 執行下列程式碼回應訊息：
+[data URL](/en-US/docs/Web/HTTP/data_URIs) 相當於網路請求，範例中的 data URL 會在 worker 執行下列程式碼回應訊息：
```
Comment (yin1999) [inline files/zh-tw/web/api/web_workers_api/using_web_workers/index.md:322]:
```suggestion
[data URL](/zh-TW/docs/Web/HTTP/data_URIs) 相當於網路請求，範例中的 data URL 會在 worker 執行下列程式碼回應訊息：
```

**Review by yin1999 — APPROVED** [review]
LGTM. Thank you @queengooborg. But let's reference the localized documents :)

**Review by sashasushko — APPROVED** [review]
Approve for `ru`. Thanks ❤️

File: files/fr/mozilla/add-ons/webextensions/sharing_objects_with_page_scripts/index.md [hunk 6]
```diff
@@ -29,7 +29,7 @@ Le but de cette fonctionnalité est de rendre le script moins privilégié plus
 
 Par exemple, lorsqu'un script de contenu accède à la [fenêtre](/fr/docs/Web/API/Window) de la page, il ne voit aucune propriété ajoutée au script de la page, et si le script de la page a redéfini les propriétés de la fenêtre, le script de contenu verra la version originale .
 
-Pour l'histoire complète sur la vision Xray, voir les articles sur [Vision Xray](en-US/docs/Mozilla/Tech/Xray_vision) et la [securité des Scripts](en-US/docs/Mozilla/Gecko/Script_security).
+Pour l'histoire complète sur la vision Xray, voir les articles sur [Vision Xray](/en-US/docs/Mozilla/Tech/Xray_vision) et la [securité des Scripts](/en-US/docs/Mozilla/Gecko/Script_security).
```
Comment (cw118) [inline files/fr/mozilla/add-ons/webextensions/sharing_objects_with_page_scripts/index.md:32]:
```suggestion
Pour l'histoire complète sur la vision Xray, voir les articles sur [Vision Xray](/fr/docs/Mozilla/Tech/Xray_vision) et la [securité des Scripts](/fr/docs/Mozilla/Gecko/Script_security).
```

File: files/fr/web/api/extendablemessageevent/index.md [hunk 7]
```diff
@@ -31,7 +31,7 @@ _Hérite des propriétés de son parent,_ _{{domxref("ExtendableEvent")}}_.
 - {{domxref("ExtendableMessageEvent.origin")}} {{readonlyinline}}
   - : Retourne l'origine du {{domxref("ServiceWorkerClient")}} qui envoie le message.
 - {{domxref("ExtendableMessageEvent.lastEventId")}} {{readonlyinline}}
-  - : Représente, dans un [server-sent events](en-US/docs/Server-sent_events/Using_server-sent_events), le dernier ID de l'évenement source.
+  - : Représente, dans un [server-sent events](/en-US/docs/Server-sent_events/Using_server-sent_events), le dernier ID de l'évenement source.
```
Comment (cw118) [inline files/fr/web/api/extendablemessageevent/index.md:34]:
```suggestion
  - : Représente, dans un [server-sent events](/fr/docs/Server-sent_events/Using_server-sent_events), le dernier ID de l'évenement source.
```

File: files/fr/web/progressive_web_apps/tutorials/js13kgames/loading/index.md [hunk 8]
```diff
@@ -9,7 +9,7 @@ original_slug: Web/Progressive_web_apps/Loading
...
 
-Dans les articles précédents, nous avons abordé les APIs qui nous aident à faire de notre exemple [js13kPWA](https://mdn.github.io/pwa-examples/js13kpwa/) une Progressive Web App: [Service Workers](en-US/docs/Web/Progressive_web_apps/Offline_Service_workers), [Manifestes Web](/fr/docs/Web/Progressive_web_apps/Installable_PWAs), [Notifications et Push](/fr/docs/Web/Progressive_web_apps/Re-engageable_Notifications_Push). Dans cet article, nous irons encore plus loin et améliorerons la performance de l'application en téléchargeant progessivement ses ressources.
+Dans les articles précédents, nous avons abordé les APIs qui nous aident à faire de notre exemple [js13kPWA](https://mdn.github.io/pwa-examples/js13kpwa/) une Progressive Web App: [Service Workers](/en-US/docs/Web/Progressive_web_apps/Offline_Service_workers), [Manifestes Web](/fr/docs/Web/Progressive_web_apps/Installable_PWAs), [Notifications et Push](/fr/docs/Web/Progressive_web_apps/Re-engageable_Notifications_Push). Dans cet article, nous irons encore plus loin et améliorerons la performance de l'application en téléchargeant progessivement ses ressources.
```
Comment (cw118) [inline files/fr/web/progressive_web_apps/tutorials/js13kgames/loading/index.md:12]:
```suggestion
Dans les articles précédents, nous avons abordé les APIs qui nous aident à faire de notre exemple [js13kPWA](https://mdn.github.io/pwa-examples/js13kpwa/) une Progressive Web App: [Service Workers](/fr/docs/Web/Progressive_web_apps/Offline_Service_workers), [Manifestes Web](/fr/docs/Web/Progressive_web_apps/Installable_PWAs), [Notifications et Push](/fr/docs/Web/Progressive_web_apps/Re-engageable_Notifications_Push). Dans cet article, nous irons encore plus loin et améliorerons la performance de l'application en téléchargeant progessivement ses ressources.
```

**Review by cw118 — APPROVED** [review]
I allowed myself to push a commit that points to fr links in instead of en-US ones ~to hopefully avoid conflicts with the localized links PR~ (never mind, I did not notice that that PR does not modify these files). That being said, thank you for this @queengooborg !

**Review by SphinxKnight — APPROVED** [review]
OK for @mdn/yari-content-fr 
Thanks for this @queengooborg :bow:

**Comment by queengooborg** [comment]
Still awaiting approval from @mdn/yari-content-pt-br and @mdn/yari-content-ja!

**Review by mfuji09 — APPROVED** [review]
LGTM for `ja`.

**Review by josielrocha — APPROVED** [review]
LGTM for `pt-BR`, thank you very much!

**Pull request merged**